packages/web/components/control/language-select.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback } from "react";
+import { FunctionComponent } from "react";
+import { useBooleanWithWindowEvent, useWindowSize } from "../../hooks";
+import { observer } from "mobx-react-lite";
+import { useStore } from "../../stores";
+import Image from "next/image";
+import { MenuDropdownIcon } from "./menu-dropdown-icon/menu-dropdown-icon";
+import { MenuDropdownIconItem } from "./types";
+import { MenuOptionsIconModal } from "../../modals";
+import { useTranslation } from "react-multi-lang";
+
+export type LanguageSelectProps = {
+  options: MenuDropdownIconItem[];
+};
+
+export const LanguageSelect: FunctionComponent<LanguageSelectProps> = observer(
```

Comment:
May pain you for me to say this, but I think we should at least change the name and make it seem generic. I posted a comment in the Figma, I think asset select is not the right term given our work with crypto assets. That way we'd be more able to reuse this component. Seems like it won't require too many logic/type changes.

Reply:
See figma comment:

https://www.figma.com/file/kYSFzlWoiMFrqkF46dtZQV?node-id=312:3892#285175175